src/avclient.jl

```diff
@@ -0,0 +1,28 @@
+const alphavantage_api = "https://www.alphavantage.co/"
+
+struct AVClient
+    key::String
+    entry::String
+end
+
+AVClient(; key = "", entry = alphavantage_api) = AVClient(key, entry)
+
+const GLOBAL = Ref(AVClient(key = haskey(ENV, "ALPHA_VANTAGE_API_KEY") ? ENV["ALPHA_VANTAGE_API_KEY"] : ""))
```

Comment:
I would use: `get(ENV, "ALPHA_VANTANGE_API_KEY, "")`, is there any specific reason for your method?

Reply:
You are absolutely right. This is just an outcome of a sequence of refactoring.